### PR TITLE
🔧 display Kodadot fee on buy

### LINKED ISSUE #8827: Show Kodadot Fee on when buying NFT
image

### PR DESCRIPTION
**Pull request by roiLeo**
- close #8827 

UPDATE
Screenshot 2024-01-06 at 14-17-57 Split #242

Screenshot 2024-01-06 at 13-45-13 Pare1d0scope #172

### CODE REVIEW

**Comment by Jarsen136** [comment]
image


Maybe percentage? code lgtm

**Review by Jarsen136 — APPROVED** [review]

**Comment by exezbcz** [comment]
@Jarsen136 here is the revised design 

https://github.com/kodadot/nft-gallery/issues/8827#issuecomment-1879764681

**Comment by roiLeo** [comment]
> @Jarsen136 here is the revised design

please check again, I don't see what's missing

**Comment by Jarsen136** [comment]
> > The design has been changed.
> 
> what's missing?
> 

Sorry, I saw the screenshot on the  PR description was old : )

File: components/common/confirmPurchaseModal/ConfirmPurchaseModal.vue [hunk 1]
```diff
@@ -28,6 +28,21 @@
           }}</span>
           <CommonTokenMoney :value="totalNFTsPrice" />
         </div>
+        <div class="flex justify-between has-text-grey is-size-7">
+          <div>
+            {{ $t('Service Commission') }}
```
Comment (Jarsen136) [inline components/common/confirmPurchaseModal/ConfirmPurchaseModal.vue:33]: translation key for it?

**Review by Jarsen136 — APPROVED** [review]

**Comment by exezbcz** [comment]
@roiLeo not much, only last detail is the spacing 

image

- could you please make it 8px and then it would be perfect! 

image

**Comment by roiLeo** [comment]
> @roiLeo not much, only last detail is the spacing
>     * could you please make it 8px and then it would be perfect!

IMO this was better before, see now:
Screenshot 2024-01-10 at 09-01-10 Pare1d0scope

**Review by preschian — APPROVED** [review]

**Pull request merged**